sdk/search/Azure.Search.Documents/samples/Sample07_VectorSearch_UsingReducedEmbeddings.md

```diff
@@ -0,0 +1,189 @@
+# Vector Search Using Reduced Embeddings
+
+This sample demonstrates how to create a vector fields index with reduced dimensions, upload reduced embeddings into the index, and query the documents. To accomplish this, you can utilize Azure OpenAI embedding models: a smaller and highly efficient `text-embedding-3-small` model or a larger and more powerful `text-embedding-3-large` model. These models are significantly more efficient and require less storage space.
+
+## Creating a Vector Index
+
+Let's consider the example of a `Hotel`. First, we need to create an index for storing hotel information. In this index, we will define vector fields called `DescriptionVector` and `CategoryVector`. To configure the vector field, you need to provide the model dimensions, which indicate the size of the embeddings generated for this field. You can pass reduced dimensions and the name of the vector search profile that specifies the algorithm configuration, along with `Vectorizer`.
+
+In order to get the reduced embeddings using either the `text-embedding-3-small` or `text-embedding-3-large` models, it is necessary to include the `Dimensions` parameter. This parameter enables customers to define the desired number of dimensions for the output vector. Therefore, for `AzureOpenAIVectorizer`, we will retrieve the `VectorSearchDimensions` that is already specified in the corresponding index field definition. However, to ensure that dimensions are only passed along in the vectorizer for a model that supports it, we need to pass a required property named `ModelName`. This property enables the service to determine which model we are using, and dimensions will only be passed along when it is for a known supported model name.
```

Comment:
```suggestion
In order to get the reduced embeddings using either the `text-embedding-3-small` or `text-embedding-3-large` models, it is necessary to include the `Dimensions` parameter. This parameter configures the desired number of dimensions for the output vector. Therefore, for `AzureOpenAIVectorizer`, we will retrieve the `VectorSearchDimensions` that is already specified in the corresponding index field definition. However, to ensure that dimensions are only passed along in the vectorizer for a model that supports it, we need to pass a required property named `ModelName`. This property enables the service to determine which model we are using, and dimensions will only be passed along when it is for a known supported model name.
```

Usage of customer in that sentence felt off as we didn't reference customer anywhere else